source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/observations.py

```diff
@@ -347,61 +375,112 @@ def __call__(
         convert_perspective_to_orthogonal: bool = False,
         model_zoo_cfg: dict | None = None,
         model_name: str = "ResNet18",
-        model_device: str | None = "cuda:0",
-        reset_model: bool = False,
+        model_device: str | None = None,
```

Comment:
Doesn't seem like model device is used here anymore (instead in constructor). Does it still need to be here?

Reply:
yes, we need this still for arg checking...

Reply:
Yes. Still needed for params cross checking.